src/checkly-handlebars.js

```diff
@@ -8,7 +8,12 @@ const create = () => {
   handlebars.registerHelper('$RANDOM_NUMBER', () => random(0, 10000))
   handlebars.registerHelper('$UUID', () => v4())
   handlebars.registerHelper(helpers(['comparison', 'date']))
-
+  handlebars.registerHelper('split', (string, separator, index) => {
+    if (index == -1) {
```

Comment:
Small usability nit: maybe default the index to `-1` so that if they don't pass an index it'll return the split array by default?

Otherwise LGTM :+1:

Reply:
Also is it purposely `==` ?

Reply:
honestly did not think about that